config/config.go

```diff
@@ -1108,13 +1140,20 @@ func (c *Config) addInput(name string, table *ast.Table) error {
 	if len(c.InputFilters) > 0 && !sliceContains(name, c.InputFilters) {
 		return nil
 	}
+
 	// Legacy support renaming io input to diskio
 	if name == "io" {
 		name = "diskio"
 	}
```

Comment:
Could it be possible this is also added in the DeprecationInfo table?

```golang
	"io": {
 		Since:  "…",
 		Notice: "use 'inputs.diskio' instead",
 	},
```

Reply:
Sure, do you also have a "since" version?